package.json

```diff
@@ -37,5 +37,15 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "husky": "^8.0.3",
+    "prettier": "3.0.0",
+    "pretty-quick": "^3.1.3"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged"
+    }
```

Comment:
cool, so this will use Prettier to format your changed files each time you commit? 🤩

Reply:
Yes, it will